README.md

```diff
@@ -78,21 +80,28 @@ $ psql
 ```
 You will see something similar to this:
 
-```
+<pre>
 psql (10.5)
 Type "help" for help.
 
-_username_=#
-```
+<i>yourusername</i>=#
+</pre>
 
-Create a new database called listings
+Create a new database called listings.
 
 <pre>
 # CREATE DATABASE listings;
-# CREATE ROLE listings WITH LOGIN PASSWORD '<i>your password not this</i>';
+# CREATE ROLE listings WITH LOGIN PASSWORD '<i>your_password_not_this</i>';
 # GRANT ALL PRIVILEGES ON DATABASE listings TO listings;
 # ALTER USER listings CREATEDB;
 </pre>
 
+In the root of your project create a file called .env and add the following:
+
```

Comment:
@chriswedgwood, one question please, do we need to add this file to .gitignore?

Reply:
I would say yes, as usually the DATABASE_URL requires the username and password for authentication.

Reply:
maybe he meant outside of the project?

Reply:
The .env is already in .gitignore under #environments :)

Reply:
oh yes, I see it now :) thanks @chriswedgwood